ILSpy/TreeNodes/NaturalStringComparer.cs

```diff
@@ -16,22 +16,37 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+using System;
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
 
+using NaturalSort.Extension;
+
 namespace ICSharpCode.ILSpy.TreeNodes
 {
+	/// <summary>
+	/// .NET natural string comparison emulating .Instance behavior of original native-interop variant
+	/// </summary>
+	public sealed class NaturalStringComparer
+	{
+		public static readonly IComparer<string> Instance = StringComparison.CurrentCultureIgnoreCase.WithNaturalSort();
+		// public static readonly NaturalStringComparer Instance = new NaturalStringComparer();
+	}
+
+#if WINDOWS
+	/// <summary>
+	/// Native-Interop natural string comparion using StrCmpLogicalW from shlwapi.dll
+	/// </summary>
 	[System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1060:MovePInvokesToNativeMethodsClass")]
-	public sealed class NaturalStringComparer : IComparer<string>
+	public sealed class NativeNaturalStringComparer : IComparer<string>
```

Comment:
Why do we keep the old one around?

Reply:
That is just for optionality reasons - eg if we ever wanted to compare for a bug report on sorting.